tensorboard/webapp/metrics/effects/index.ts

```diff
@@ -42,14 +42,18 @@ import {
   TagMetadata,
   TimeSeriesRequest,
   TimeSeriesResponse,
-} from '../data_source';
+} from '../data_source/index';
```

Comment:
Without this, requirejs fails at runtime with
`There is no timestamp for org_tensorflow_tensorboard/tensorboard/webapp/metrics/data_source.js!`